creme/stats.py

```diff
@@ -205,4 +205,26 @@ def update(self,x):
         return self 
     
     def get(self):
-        return self.max.get() - self.min.get()
\ No newline at end of file
+        return self.max.get() - self.min.get()
+    
+class Sum(RunningStatistic):
+    """Computes a running Sum.
+    
+    Attributes:
+        sum (float) : The running sum.
+
+    """
+    def __init__(self):
+        super().__init__()
+        self.sum = 0.0
+    
+    @property
+    def name(self):
+        return 'sum'
+    
+    def update(self,x):
```

Comment:
You just missed a space after the comma here :). Apart from that everything seems fine.